doc/admin-guide/files/records.config.en.rst

```diff
@@ -3901,6 +3901,19 @@ HTTP/2 Configuration
    Clients that send smaller window increments lower than this limit will be immediately disconnected with an error
    code of ENHANCE_YOUR_CALM.
 
+.. ts:cv:: CONFIG proxy.config.http2.write_buffer_block_size_index INT 11
```

Comment:
Doing an index size seems weird.  I think it would be better to specify the buffer size and say we round up or down based on a power of 2.  Look to see if there is another config option or storage.config that does something similar.